styles/main.scss

```diff
@@ -1481,3 +1481,170 @@ blockquote p:last-child {
 .posts-list-block:hover .post-img:after {
   height: 100%;
 }
+
+/*audio-player ==============================================*/
+
+.audio {
+  position: relative;
+  width: 100%;
+  height: 53px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 16px 7px 10px;
+  background: #f2f5f7;
+  border-radius: 4px;
+  margin-bottom: 18px;
+}
+
+.audio button {
+  border: none;
+  cursor: pointer;
+  padding: 0;
+}
+
+.audio button .pause-icon {
+  object-fit: cover;
+}
+
+.audio .play {
+  position: relative;
+  width: 36px;
+  height: 33px;
+  background: #ffffff;
+  box-shadow: 0px 4px 16px rgba(0, 0, 0, 0.12);
+  border-radius: 50%;
+  flex: 1 0 33px;
+}
+
+.audio .play img {
+  position: absolute;
+  left: 54%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.audio .volume {
+  background-color: transparent;
+  margin-bottom: 2px;
+}
+
+.audio .volume-bar {
+  opacity: 0;
+  pointer-events: none;
+
+  position: absolute;
+  display: flex;
+  width: 157px;
+  height: 38px;
+  padding: 13px;
+
+  transform: rotate(-90deg);
+  bottom: 120px;
+  right: -60px;
+
+  background: #f2f5f7;
+  border-radius: 4px;
+
+  justify-content: center;
+  align-items: center;
+}
+
+.audio .volume-bar[closing] {
+  animation: fade-out 500ms forwards;
+}
+
+.audio .volume-bar[open] {
+  animation: fade-in 500ms forwards;
+}
+
+.audio .volume-bar input {
+  height: 4px;
+  width: 100%;
+
+  --seek-before-width: 0;
+
+  background: #ffffff;
+
+  appearance: none;
+  border-radius: 4px;
+  position: relative;
+  outline: none;
+  cursor: pointer;
+}
+
+.audio .volume-bar input::before {
+  content: "";
+  height: 4px;
+  width: var(--seek-before-width);
+
+  background: #000000;
+  border-radius: 4px;
+  position: absolute;
+  z-index: 2;
+}
+
+.audio .volume-bar input::-webkit-slider-thumb {
+  opacity: 0;
+}
+
+.audio .progress-bar {
+  margin-left: 9px;
+  margin-right: 18px;
+  width: 100%;
+}
+
+.audio .progress-bar input {
+  --seek-before-width: 0;
+
+  appearance: none;
+  background: #ffffff;
+  border-radius: 4px;
+  position: relative;
+  width: 100%;
+  height: 4px;
+  outline: none;
+  cursor: pointer;
```

Comment:
How you can notice, two inputs have some similar styles. What do you think about combine similar styles without repeating them?🧐